recipes/biopython/meta.yaml

```diff
@@ -4,17 +4,17 @@ package:
 
 about:
   home: http://www.biopython.org/
-  license: MIT
+  license: Biopython License Agreement
+  license_file: LICENSE
   summary: 'Freely available tools for computational molecular biology.'
-  license_family: MIT
 
 source:
   fn: biopython-1.68.tar.gz
   url: https://pypi.python.org/packages/72/6c/e1e13b9df73f9c2539b67d12bc22be6b19779230cadbed04c24f3f3e5ef4/biopython-1.68.tar.gz
   md5: 078e915185485a5327937029b7577ddc
 
 build:
-  number: 0
+  number: 1
 
```

Comment:
@peterjc would you please add a skip for py34 as [here](https://github.com/bioconda/bioconda-recipes/pull/4403/files#diff-0d6ff46815d4d9860d398f6a9f1507cfR23)? Might also need the `numpy x.x` as [here](https://github.com/bioconda/bioconda-recipes/pull/4403/files#diff-0d6ff46815d4d9860d398f6a9f1507cfR28). Hopefully that'll get this to pass.